dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -244,4 +251,90 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             LOG_IF_FAILED(::CoRevokeClassObject(static_cast<DWORD>(token.Cookie())));
         }
     }
+
+    bool PushNotificationManager::IsBIAvailable()
+    {
+        BackgroundTaskBuilder builder = BackgroundTaskBuilder();
+        try
+        {
+            auto builder5 = builder.as<winrt::IBackgroundTaskBuilder5>();
+        }
+        catch (...)
+        {
+            return false;
+        }
+        return true;
+    }
+
+    bool PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationOptions const& options)
+    {
+        if (!WI_AreAllFlagsClear(options, PushNotificationRegistrationOptions::PushTrigger | PushNotificationRegistrationOptions::ComActivator |
```

Comment:
Do we have a WI_* function that can tell you if the variable is zero essentially without passing all flags? Can you double check?

Reply:
WI_IsSingleFlagSet checks if a single bit is set.